### ISSUE TITLE
v2.x  Better display of the selected "multipled" marked widget

### ISSUE THREAD

**Issue by dos1973** [ioBroker/ioBroker.vis#584]
**Describe the bug**  
Better display of multipled selected widget
Better display of marked or selected widgets. With v2 only the anchor points are highlighted...

**To Reproduce**  
mark a couple of widgets, the recognition of the widgets which are  marked to be edited is difficult to recognize,

**Expected behavior**  
no better idea than adding addional the dotted line ob every marked widget

**Screenshots & Logfiles**  
[iobroker forum](https://forum.iobroker.net/topic/60398/test-adapter-vis-2-0-alpha/77?_=1669663252664)

**Versions:**  
v2.08

**Comment by Feuer-sturm** [comment]
I agree. With dark background the blue line of one selected item is hard to see on the whole screen
![grafik](https://user-images.githubusercontent.com/25422320/204373650-79b012e7-8802-466d-afe2-a31e1282184f.png)
In vis-1 the dotted lines where white and blue which is in my oppinion a better contrast

If at minimum 2 elements are selcted the dotted lines are not available any more
![grafik](https://user-images.githubusercontent.com/25422320/204373991-a260ffff-fd4a-40f1-8c3e-1c109f3670dd.png)
